### ISSUE TITLE
Integration to SMART

### ISSUE THREAD

**Issue by clapierre** [daisy/ace-gui#22]
Would love to see a one click button to send the JSON file to SMART for continued manual evaluation

**Comment by rdeltour** [comment]
Right, finer integration with SMART is definitely something to think about, and a "send to SMART" simple command would be a great first step, thanks for the idea!

Not everyone will have a SMART access however, so maybe we can disable it and enable it via preferences? Let's figure this out.

**Comment by mattgarrish** [comment]
You have to authenticate to the site before you can use the tool, so the user will probably have to store credentials. A case for using the preferences to determine if they have access.

Since you're already using a web viewer already, could you integrate smart into the tool (i.e., if they store their information, include a tab with the interface loaded into it when initiated). You could then post the data to the evaluation page in the tab.
